src/App.tsx

```diff
@@ -1,35 +1,19 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Home from "./pages/Home";
+import StartGame from "./pages/StartGame";
 
 function App() {
-  const [count, setCount] = useState(0)
+	return (
```

Comment:
Just a thought for ourselves for the future: we might want to extract the Router-logic to it's own component in the future, i.e simplify App to something like:

```
function App() {
   return <PageRouter />;
}
```

And let `PageRouter` contain all Router-specific logic, like imports from `'react-router-dom' and our page-components. This will be nice to have if our page's router-logic grows in the future. Also, if we add context-providers in the future that wraps all of our routes it will be nice if App grow beyond something like:

```
function App() {
   return (
      <DataProvider>
         <PageRouter />;
      </DataProvider>
   )
}
```

But for now, since we don't have that many routes or providers, this works great.